.github/workflows/ci-snapshot.yml

```diff
@@ -0,0 +1,45 @@
+name: Snapshot Generation
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  create-installer:
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v1
+    - run: |
+        git checkout -b github_actions_build_branch___ # have to be on a branch
```

Comment:
Is there an actions variable for the branch that was pushed?

Reply:
Ah, good to know. Then I can fix that to checkout the correct branch.

The problem is that snapshot building clones and then pulls from the repository that you start with, which doesn't work in "detached head" mode. So we need to be on some branch.